src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -171,20 +150,13 @@ limitations under the License.
         <profileId editable="false">0x0103</profileId>
         <deviceId editable="false">0x000e</deviceId>
```

Comment:
This device type is called Aggregator in the spec, not Bridge.

Reply:
Updated in seperate pr